app/test/socobo-inventory/socobo-inventory-create.html

```diff
@@ -66,7 +67,9 @@
       });
 
       test('create button is hidden when in edit mode', function(done) {
-        dialog.show({});
+        var item = {measurement: {value: 'pcs'}, desc: 'test item', info: 'test info'};
```

Comment:
@lyio You was right! We don't have to iterate over the measurements if we use indexOf!!! I couldn't fix this because the pull request was already merged.

@lyio and @flashback2k14 First I think storing the measurements in Object violates KISS, but that is secondly. More important is the fact that using Objects breaks the _resetItem method because the indexOf doesn't work anymore. Because of that the measurements aren't displayed in the edit dialog.
To fix that we need to change the resetItem method. I will open a pull request for that.